packages/crud/test/dom/__snapshots__/crud.test.snap.js

```diff
@@ -148,33 +143,6 @@ snapshots["vaadin-crud host default"] =
   >
     New item
   </vaadin-button>
-  <vaadin-button
```

Comment:
Why were these buttons removed?

Reply:
See https://github.com/vaadin/web-components/pull/6232#discussion_r1273456315. Previously, `_form` was set after `_fullscreen` property, so initial nodes were not teleported on `_fullscreen` change. Now, `_form` is set by the controller and this check returns `true`.

https://github.com/vaadin/web-components/blob/0a42994dfde006ea89aad081418eca0310e6efd8/packages/crud/src/vaadin-crud.js#L867-L871